deepchecks/utils/distribution/drift.py

```diff
@@ -131,23 +131,23 @@ def cramers_v(dist1: Union[np.ndarray, pd.Series], dist2: Union[np.ndarray, pd.S
     """
     # If balance_classes is True, min_category_size_ratio should not affect results:
     min_category_size_ratio = min_category_size_ratio if balance_classes is False else 0
-
-    dist1_counts, dist2_counts, _ = preprocess_2_cat_cols_to_same_bins(dist1, dist2, min_category_size_ratio,
-                                                                       max_num_categories, sort_by)
-
+    dist1_counts, dist2_counts, cat_list = preprocess_2_cat_cols_to_same_bins(dist1, dist2,
+                                                                              min_category_size_ratio,
+                                                                              max_num_categories, sort_by)
+    if len(cat_list) == 1:  # If the distributions have the same single value
+        return 0
     if balance_classes is True:
         dist1_counts, dist2_counts = rebalance_distributions(dist1_counts, dist2_counts)
 
-    contingency_matrix = pd.DataFrame([dist1_counts, dist2_counts])
-
-    # If columns have the same single value in both (causing division by 0), return 0 drift score:
-    if contingency_matrix.shape[1] == 1:
-        return 0
+    # Down sample one of the distributions so that they have the same total number of samples
+    dist1_sum, dist2_sum = sum(dist1_counts), sum(dist2_counts)
```

Comment:
Do this only if not balance_classes ?

Reply:
We will handle unbalanced after, but gut feeling is it is still required